pgcli/packages/sqlcompletion.py

```diff
@@ -22,6 +22,7 @@
 Schema = namedtuple('Schema', [])
 Table = namedtuple('Table', ['schema'])
 JoinCondition = namedtuple('JoinCondition', ['tables', 'parent'])
```

Comment:
Can you add a quick comment here explaining the distinction between Join and JoinCondition?